stone/backends/obj_c_types.py

```diff
@@ -1120,7 +1120,7 @@ def emit_serializer():
                         if is_user_defined_type(data_type):
                             if is_struct_type(data_type) and \
                                     not data_type.has_enumerated_subtypes():
-                                self.emit('jsonDict = [{} mutableCopy];'.
```

Comment:
this caused the jsonDict to become nil which prevented a .tag field from being added